character-physics.js

```diff
@@ -55,15 +54,20 @@ class CharacterPhysics {
     this.characterWidth = characterWidth;
     this.characterHeight = characterHeight;
 
-    const radius = (this.characterWidth / CHARACTER_CONTROLLER_HEIGHT_FACTOR) * this.characterHeight;
-    const height = this.characterHeight - radius * 2;
+    const radius = this.characterWidth / 2;
+    let height = this.characterHeight - radius * 2;
 
-    const contactOffset = (0.1 / CHARACTER_CONTROLLER_HEIGHT_FACTOR) * this.characterHeight;
-    const stepOffset = (0.5 / CHARACTER_CONTROLLER_HEIGHT_FACTOR) * this.characterHeight;
+    // avatar.height is the height of eye from the floor
+    // convert to full height by adding some more
+    const heightOffset = radius * 1.5;
+    this.yOffset = -heightOffset / 2; // add this after physx cct position output
+    height = height + heightOffset; 
+
+    const contactOffset = 0.1 * this.characterHeight;

```

Comment:
Shouldn't these be constants that align with the PhysX implementation?

Reply:
Actually never mind, I see in the diff unroll it's later used in construction 👍.